src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -53,16 +54,38 @@ public static void OnPostProcessBuild(BuildTarget target, string executablePath)
             }
         }
 
-        private static void AddCrashHandler(IDiagnosticLogger logger, string projectDir)
+        private static bool IsEnabledForPlatform(BuildTarget target, SentryUnityOptions options) => target switch
         {
-            var crashpadPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins",
-                "Windows", "Sentry", "crashpad_handler.exe"));
-            var targetPath = Path.Combine(projectDir, Path.GetFileName(crashpadPath));
-            logger.LogInfo("Copying the native crash handler '{0}' to the output directory", Path.GetFileName(crashpadPath));
+            BuildTarget.StandaloneWindows64 => options.WindowsNativeSupportEnabled,
```

Comment:
```suggestion
            BuildTarget.StandaloneWindows => options.WindowsNativeSupportEnabled,
            BuildTarget.StandaloneWindows64 => options.WindowsNativeSupportEnabled,
```
Is this missing?

Reply:
no, we don't support 32-bit windows. It was there originally by mistake